test/y2storage/callbacks/callbacks_examples.rb

```diff
@@ -19,6 +19,8 @@
 # find current contact information at www.suse.com.
 
 RSpec.shared_examples "general #error examples" do
+  ERROR_EXAMPLES_MAX_LENGTH = 80
```

Comment:
Now that we are touching this... there is any good reason for it to be defined as a constant instead of just using a variable or a `let`?

Reply:
I do not think so, just wanted to keep the original "intention" (if any). A `let` is OK for me (and a constant too).